app/lib/sql.py

```diff
@@ -1,8 +1,7 @@
 import asyncio
 from collections import namedtuple
-from typing import Annotated
 
-from fastapi import File, UploadFile
+from fastapi import UploadFile
 
 from app.models import GetMembers, MemberIn, Category, CategoryOut, MemberWithCategory, MemberHasCategoryIn, \
```

Comment:
Why not using imports with parentheses ? 
```py
from app.models import (
    MemberIn,
    MemberOut,
    Category,
    CategoryOut,
    MemberWithCategory,
    MemberHasCategoryIn,
    GetMemberHasNetwork,
    MemberById,
    Network,
    MemberHasNetwork,
    MemberHasCategory,
    MemberHasNetworkIn
)
```

or even better, 
```py
from app import models
```
and in the code, for example 

```py
@app.get("/members", response_model=List[models.MemberWithCategory])
async def api_get_members():
    return await get_members()

```